Submitted By: Jim Gifford (patches at jg555 dot com)
Date: 2003-09-10
Initial Package Version: 1.4.2
Origin: inetutils-1.4.2
Description: Change int to socklen since it's a socket call

diff -Naur inetutils-1.4.2.orig/telnetd/telnetd.c inetutils-1.4.2/telnetd/telnetd.c
--- inetutils-1.4.2.orig/telnetd/telnetd.c	2002-06-26 03:15:07.000000000 +0000
+++ inetutils-1.4.2/telnetd/telnetd.c	2003-09-10 16:46:15.000000000 +0000
@@ -270,12 +270,14 @@
 
 typedef unsigned int ip_addr_t; /*FIXME*/
 
+extern char *localhost __P ((void));
+
 void
 telnetd_setup (int fd)
 {
   struct sockaddr_in saddr;
   int true = 1;
-  int len;
+  socklen_t len;
   struct hostent *hp;
   char uname[256]; /*FIXME*/
   int level;
